### PR TITLE
feat:cart screen added

### PR DESCRIPTION
**Pull request by rajkishancode**

### CODE REVIEW

File: screens/cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+.cart-content {
+    gap: 4.8rem;
+}
+
+.cart-details {
+    max-width: 52rem;
+}
+
+.cart-details {
+    background: var(--auth-background);
```
Comment (ckahub) [inline screens/cart/cart.css:23]: use rem everywhere

**Review by ckahub — COMMENTED** [review]
lgtm

File: screens/cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+
+.btn {
+    width: 100%;
+    padding: 1.5rem 0;
+    margin: 1rem 0;
+}
+
+.cart .card-image {
+    width: 28.8rem;
```
Comment (ianuragsharma) [inline screens/cart/cart.css]: giving fix width might lead to responsiveness issue,max-width might be a good option

**Review by ianuragsharma — APPROVED** [review]
looks good

File: screens/cart/cart.css (hunk 2 above)
Comment (rajkishancode) [reply screens/cart/cart.css]: Done

**Pull request merged**